django/contrib/admin/sites.py

```diff
@@ -176,17 +177,16 @@ def check_dependencies(self):
         """
         Check that all things needed to run the admin have been correctly installed.
 
-        The default implementation checks that admin and contenttypes apps are
-        installed, as well as the auth context processor.
+        The default implementation checks that the contenttypes app is
+        installed, as well as the auth context processor only if using
+        the default authentication backend.
         """
-        if not apps.is_installed('django.contrib.admin'):
-            raise ImproperlyConfigured(
-                "Put 'django.contrib.admin' in your INSTALLED_APPS "
-                "setting in order to use the admin application.")
         if not apps.is_installed('django.contrib.contenttypes'):
-            raise ImproperlyConfigured(
-                "Put 'django.contrib.contenttypes' in your INSTALLED_APPS "
-                "setting in order to use the admin application.")
+            missing_app = Error(
+                "'django.contrib.contenttypes' must be in INSTALLED_APPS "
+                "in order to use the admin application'",
+                id="admin.E401")
```

Comment:
While you are touching this, you can convert this to use our preferred indent style of putting the closing parenthesis on the next line.